src/openrct2/object/ObjectFactory.cpp

```diff
@@ -251,9 +251,10 @@ namespace ObjectFactory
         }
     }
 
-    std::unique_ptr<Object> CreateObjectFromLegacyFile(IObjectRepository& objectRepository, const utf8* path, bool loadImages)
+    std::unique_ptr<Object> CreateObjectFromLegacyFile(
+        IObjectRepository& objectRepository, const std::string& path, bool loadImages)
```

Comment:
shouldn't we be using u8string for this?

Reply:
I suppose